### ISSUE TITLE
Should it depend on Biopython

### ISSUE THREAD

**Issue by sjackman** [kblin/ncbi-acc-download#2]
`pip3 install ncbi-acc-download` installed and succeeded, but it didn't install `biopython`.
I ran `ncbi-acc-download NC_005278.1` and got the error message…
```
Traceback (most recent call last):
  File "/Users/sjackman/.homebrew/bin/ncbi-acc-download", line 7, in <module>
    from ncbi_acc_download.__main__ import main
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/ncbi_acc_download/__main__.py", line 6, in <module>
    from .core import download_to_file, Config, HAVE_BIOPYTHON
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/ncbi_acc_download/core.py", line 24, in <module>
    from ncbi_acc_download.validate import HAVE_BIOPYTHON, run_extended_validation, VALIDATION_LEVELS
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/ncbi_acc_download/validate.py", line 7, in <module>
    from Bio import SeqIO
ModuleNotFoundError: No module named 'Bio'
```
Should `setup.py` specify that `ncbi-acc-download` depends on `biopython`?

**Comment by kblin** [comment]
Huh? Did Python 3.7 change from `ImportError` to `ModuleNotFoundError`? Darn...

BioPython is an optional dependency for when you want the extended biopython-based validation. I have an extra import try/except dance in validate.py for that: https://github.com/kblin/ncbi-acc-download/blob/master/ncbi_acc_download/validate.py#L6

But I'm catching `ImportError`, which works fine on my machine :tm:. I'll see if I can reproduce.

**Comment by kblin** [comment]
Should be fixed in 0.1.2
